internal/pkg/scalable/scaledobjects.go

```diff
@@ -14,7 +15,7 @@ const (
 	annotationKedaPausedReplicas = "autoscaling.keda.sh/paused-replicas"
 )
 
-// getScaledObjects is the getResourceFunc for Keda ScaledObjects.
+// getScaledObjects is the getResourceFunc for Keda ScaledObjects. //nolint:dupl.
```

Comment:
do we still need this no lint here if we have the whole file nolinted fo rthis linter

```suggestion
// getScaledObjects is the getResourceFunc for Keda ScaledObjects.
```